src/v3/README.md

```diff
@@ -1,12 +1,152 @@
+| Flows | Profile enrollment</br>Authenticator enrollment</br>Authenticator verification</br>Forgot password</br>Account unlock | Profile enrollment</br>Authenticator enrollment</br>Authenticator verification</br>Forgot password</br>Account unlock |
+| Globalization | Right-to-left languages (requires customization)</br>All out-of-the-box translations</br>String customization</br>Bring your own language | All out-of-the-box translations</br>String customization</br>Bring your own language |
+| Branding | Branding and multibrands</br>CSS overrides (code editor) | Branding and multibrands |
+
+
+## Type of Applications
+
+### OIDC
+
+* SPA Introduction
+* Web Introduction
+
+### Non-OIDC
+
+* SAML?
+
+
+## Okta deployment models
+
+### Redirect (Okta-hosted)
+
+A user sign-in flow that grants authentication control to Okta by redirecting to an Okta hosted sign-in page using open protocols like OAuth 2.0 and SAML. See [Redirect authentication](https://developer.okta.com/docs/guides/redirect-authentication/).
+
+### Embedded (self-hosted)
+
+The third generation Sign-In Widget doesn't support embedded authentication. See [Embedded (self-hosted), second generation](https://github.com/okta/okta-signin-widget#embedded-self-hosted).
+
+## Migrating to the third generation
+
+To migrate from a previous generation widget, first ensure that your org uses the Okta Identity Engine. All new orgs created after March 1, 2022 are Identity Engine orgs.
+
+To determine whether your org uses Classic or Identity Engine, check the Admin Console footer. The bottom of every page includes a version string that looks like `Version 2023.04.1 E`.
+
+`C` means your org is Classic; `E` means Identity Engine.
+
+* If your org is Classic, you need to upgrade to Identity Engine before migrating to the latest widget. [Learn more about upgrading to Identity Engine.](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/oie-upgrade-eligibility.htm)
+
+* If your org uses Identity Engine, you're ready to plan your widget migration.
+
+Start by evaluating whether the 3rd-generation widget supports everything you need. TODO supported capabilities
+
+TODO start with proof of concept, test org
+
+TODO enable EA
+
+TODO check brand color contrast
+
+TODO check sign-in page widget version
+
+### Restore the second generation widget
+
+If you enable the third generation widget and find that it doesn't work for your org yet, you can restore the second generation.
+
+1. In the Admin Console, go to **Settings** > **Features**.
+
+1. Select the **Sign-In Widget third generation** toggle to turn it off.
+
+1. Click **Save**.
+
+## Advanced
```

Comment:
@lesterchoi-okta Can you tell me if the README needs these four sections (Advanced to Pre-requisistes)?